src/bootstrap/doc.rs

```diff
@@ -69,6 +69,27 @@ book!(
     RustdocBook, "src/doc/rustdoc", "rustdoc", RustbookVersion::MdBook1;
 );
 
+fn generate_jump_version_js(p: &Path) -> String {
+    let content = fs::read(p).expect("file not found");
+    let s = String::from_utf8_lossy(&content);
+    let version_parts = crate::channel::CFG_RELEASE_NUM.split(".").collect::<Vec<_>>();
+    let mut versions = vec!["\"nightly\"".to_owned(),
+                            "\"beta\"".to_owned(),
+                            "\"stable\"".to_owned()];
+    let mut middle_version = i32::from_str_radix(version_parts[1], 10).expect("unknown number");
+
+    while middle_version >= 0 {
+        versions.push(format!("\"1.{}.0\"", middle_version));
```

Comment:
would be cool if we could also add bugfix releases in here, but that might be hard

Reply:
Bugfixes don't change API normally so I'm not sure it's really useful to add them in here.